tests/unit/mongo.spec.ts

```diff
@@ -2485,6 +2485,74 @@ describe('Pipeline to mongo translator', () => {
     ]);
   });
 
+  it('can generate a join step with column names containing any character', () => {
+    const rightPipeline: Pipeline = [
+      { name: 'domain', domain: 'right' },
+      { name: 'delete', columns: ['useless'] },
+    ];
+    const pipeline: Pipeline = [
+      {
+        name: 'domain',
+        domain: 'test',
+      },
+      {
+        name: 'rename',
+        oldname: 'old',
+        newname: 'new',
+      },
+      {
+        name: 'join',
+        right_pipeline: rightPipeline,
+        type: 'inner',
+        on: [
+          ['some column', 'some column'],
+          ['other column', 'other column'],
+        ],
+      },
+    ];
+    const querySteps = mongo36translator.translate(pipeline);
+    expect(querySteps).toEqual([
```

Comment:
```suggestion
    expect(querySteps).toStrictEqual([
```
doesn't cost anything as well